src/devices/legrand.ts

```diff
@@ -722,7 +722,33 @@ export const definitions: DefinitionWithExtend[] = [
         model: "281506",
         vendor: "Legrand",
         description: "Triple light switch",
-        extend: [m.deviceEndpoints({endpoints: {right: 1, center: 2, left: 3}}), m.onOff({endpointNames: ["right", "center", "left"]})],
+        ota: true,
+
+        fromZigbee: [fz.identify, fz.legrand_binary_input_on_off, fzLegrand.cluster_fc01],
+
+        toZigbee: [tzLegrand.identify, tzLegrand.led_mode],
+
+        exposes: [
+            e.switch().withEndpoint("right"),
```

Comment:
the `e.switch()` should not be needed here, it's already exposed through `modernExtend.onOff({`